examples/scaling-criteo/02-ETL-with-NVTabular.ipynb

```diff
@@ -351,7 +351,11 @@
     "categorify_op = Categorify(out_path=stats_path, max_size=num_buckets)\n",
```

Comment:
<p>are these raw parquet files? are you processing all 24 parquet files in this notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>